src/main/java/com/diamondmarket/orders/controllers/OrderController.java

```diff
@@ -0,0 +1,143 @@
+		} catch (Exception e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+			return errorResponse(context, e, HttpStatus.BAD_REQUEST);
+		}
+	}
+	
+	@RequestMapping(value = "v1/orders/deleteOrders", produces = "application/json" , method = RequestMethod.DELETE)
+	public ResponseEntity<Response> deleteOrders(@RequestHeader HttpHeaders httpHeaders, @RequestParam("orderId") String orderId) {
+		
+		TransactionContext context = new TransactionContext();
+		if(httpHeaders.get("correlationId") != null) {
+			context.setCorrelationId(httpHeaders.get("correlationId").toString());	
+		} else {
+			context.setCorrelationId("demo");
+		}
+		if(httpHeaders.get("ApplicationLabel") != null) {
+			context.setApplicationLabel(httpHeaders.get("ApplicationLabel").toString());
+		} else {
+			context.setApplicationLabel("demo");
+		}
+		
+		try {
+			deleteOrdersService.deleteOrders(context, orderId);
+			return successResponse(context, "user orders are successfully deleted with userId " + orderId, HttpStatus.OK);
+		} catch (Exception e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+			return errorResponse(context, e, HttpStatus.BAD_REQUEST);
+		}
+	}
+	
+	@RequestMapping(value = "v1/orders/placeOrder", produces = "application/json" , method = RequestMethod.POST)
+	public @ResponseBody ResponseEntity<Response> placeOrder(@RequestHeader HttpHeaders httpHeaders, @RequestBody InputRequest inputRequest) {
+		
+		TransactionContext context = new TransactionContext();
+		if(httpHeaders.get("correlationId") != null) {
+			context.setCorrelationId(httpHeaders.get("correlationId").toString());	
+		} else {
+			context.setCorrelationId("demo");
+		}
+		if(httpHeaders.get("ApplicationLabel") != null) {
+			context.setApplicationLabel(httpHeaders.get("ApplicationLabel").toString());
+		} else {
+			context.setApplicationLabel("demo");
+		}
+		
+		try {
+			return successResponse(context, placeOrderService.placeOrder(inputRequest), HttpStatus.OK);
+		} catch (Exception e) {
+			// TODO: handle exception
+			e.printStackTrace();
+			return errorResponse(context, e, HttpStatus.BAD_REQUEST);
+		}
+	}
+	
+	private ResponseEntity<Response> successResponse(TransactionContext context, Object object, HttpStatus httpStatus){
+		HttpHeaders headers = new HttpHeaders();
+		headers.add("correlationId", context.getCorrelationId());
+		headers.add("ApplicationLabel", context.getApplicationLabel());
+		headers.add("Content-Type", "application/json");
+		Response response = new Response();
+		response.setData(object);
+		response.setTimeStamp(new SimpleDateFormat("yyyy.MM.dd.HH.mm.ss").format(new Date()));
+		
+		ResponseEntity<Response> responseEntity = new ResponseEntity<Response>(response, headers , httpStatus);
+		return responseEntity;
+	}
+	
+	private ResponseEntity<Response> errorResponse(TransactionContext context, Exception exception, HttpStatus httpStatus){
+		HttpHeaders headers = new HttpHeaders();
```

Comment:
success response and error response, both the method has redundant code

Reply:
i am not doing this as both headers may have different settings